dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/GrantInfo.java

```diff
@@ -0,0 +1,67 @@
+package edu.harvard.iq.dataverse.export.openaire;
+
+/**
+ * Class which holds information's required to generate OpenAire funder contributor.
+ */
+public class GrantInfo {
+
+    private String grantFunder;
+    private String grantFunderShort;
+    private String grantProgram;
+    private String grantId;
+
+    // -------------------- GETTERS --------------------
+
+    public String getGrantFunder() {
+        return grantFunder;
+    }
+
+    public String getGrantFunderShort() {
+        return grantFunderShort;
+    }
+
+    public String getGrantProgram() {
+        return grantProgram;
+    }
+
+    public String getGrantId() {
+        return grantId;
+    }
+
+    // -------------------- SETTERS --------------------
+
+    public void setGrantFunder(String grantFunder) {
+        this.grantFunder = grantFunder;
+    }
+
+    public void setGrantFunderShort(String grantFunderShort) {
+        this.grantFunderShort = grantFunderShort;
+    }
+
+    public void setGrantProgram(String grantProgram) {
+        this.grantProgram = grantProgram;
+    }
+
+    public void setGrantId(String grantId) {
+        this.grantId = grantId;
+    }
+
+    // -------------------- LOGIC --------------------
+
+    public boolean areAllFieldsPresent() {
+        return grantFunder != null && grantFunderShort != null && grantId != null && grantProgram != null;
```

Comment:
Technically if user inputs nothing there will be null since that's how jsf does it, but in terms of future changes it could be safer so I will change it.